lib/sles4sap/azure_cli.pm

```diff
@@ -215,6 +257,36 @@ sub az_network_publicip_create {
     assert_script_run($az_cmd);
 }
 
+
+=head2 az_network_publicip_get
+
+    az_network_publicip_get(
+        resource_group => 'openqa-rg',
+        name => 'openqa-pip')
+
+Return an IPv4 public IP address from its name
+
+=over 5
+
+=item B<resource_group> - existing resource group where to create the PubIP
```

Comment:
Is the description correct? It says 'where to create IP addr', but the function does not create anything.